src/javascript/Shopper/ShopNav/index.module.scss

```diff
@@ -0,0 +1,37 @@
+.dealerHero {
+  @apply flex flex-col items-center justify-center h-full;
+}
+
+.DealerRegistration,
+.DealerLogin {
+  @apply flex items-center min-h-full;
+}
+
+.formContainer {
+  @apply max-w-xl mx-auto space-y-6 py-4 w-full shadow-lg px-8 py-12 rounded-md bg-white;
+
+  > .formTitle {
+    @apply text-center text-2xl font-medium;
+  }
+
+  > .formControls {
+    @apply space-y-6;
+  }
+}
+
+.formSection {
+  @apply grid grid-cols-6 gap-3;
+
+  > .formItem,
+  .formInput {
+    @apply h-full;
+  }
+}
+
+.submitButton {
+  @apply max-w-md w-full mx-auto flex justify-center py-2 px-4 border border-transparent text-sm font-medium rounded-md text-white bg-blue-600;
+
+  &.pending {
+    @apply bg-blue-600 text-gray-200 opacity-80;
+  }
+}
```

Comment:
Are these styles being used at all in the ShopNav at all? If not, can we just remove the file.

Reply:
Nope Tried to copy did not implement correctly.